pkg/fanal/artifact/vm/ebs.go

```diff
@@ -110,7 +110,7 @@ func (a *EBS) calcCacheKey(key string) (string, error) {
 }
 
 func (a *EBS) hasCache(cacheKey string) bool {
```

Comment:
It might make sense to add a context to this function, because the Inspect function has a context.